cal/cal/templates/home_logged_in.html

```diff
@@ -80,13 +81,53 @@ <h4 class="well">Categories - Last Week</h4>
       </ul>
       <nvd3 options="options" data="categories"></nvd3>
     </div>
-    <div ng-controller="TagsCtrl">
+    <div ng-controller="TagsCtrl as $ctrl">
       <h4 class="well">Tags - Cumulative</h4>
-      <ul class="list-group">
-        <li class="list-group-item" ng-repeat="tag in tags">
-          {% verbatim %} {{ tag.label }}: {{ tag.hours }} hours {% endverbatim %}
-        </li>
-      </ul>
+        <table class="table table-hover">
+          <thead>
+            <th>Label</th>
+            <th>Hours</th>
+          </thead>
+          <tbody>
+            <tr ng-repeat="tag in tags">
+              {% verbatim %}
+              <td ng-show="!tag.showEdit">{{ tag.label }}</td>
+              <td ng-show="!tag.showEdit">{{ tag.hours }}</td>
+              {% endverbatim %}
+              <td ng-show="tag.showEdit">
+                Label<br><input type="text" ng-model="tag.new_label">
```

Comment:
Can we have the input on the next line, since we're using a `<br>`? Happens a few other places as well